src/redux/actions/delete-kb-credentials-failure.js

```diff
@@ -1,6 +1,6 @@
 export const DELETE_KB_CREDENTIALS_FAILURE = 'DELETE_KB_CREDENTIALS_FAILURE';
 
-export const deleteKBCredentialsFailure = ({ errors }) => ({
+export const deleteKBCredentialsFailure = error => ({
```

Comment:
Please check if error messages are displayed correctly.

In `/epics/delete-kb-credentials.js` we're passing an object with `errors` property to action as payload
```
.catch(errors => Observable.of(deleteKBCredentialsFailure({ errors })));
```
so in either an action or a reducer we should destructure this object to actually get an array of errors. If we don't then in store we'll have structure like so:
```
isLoaded: false,
isFailed: true,
...
errors: {
  errors: [...]
}
```